source/manual/alerts/renew-tls-certificate.html.md

```diff
@@ -14,24 +14,28 @@ These checks look at the validity of the TLS certificates for:
 * www.integration.publishing.service.gov.uk at the edge (Fastly)
 * \*.publishing.service.gov.uk, \*.staging.publishing.service.gov.uk and \*.integration.publishing.service.gov.uk at the origin (our servers), depending on the environment Icinga is running in
 
-You'll start seeing an alert 30 days before the relevant certificate is due to
-expire.
+The alert fires 30 days before certificate expiry.
 
-See [renew a TLS certificate for GOV.UK](/manual/renew-a-tls-certificate.html)
-for details of how to renew the relevant certificate. This is normally done by
-GOV.UK Replatforming.
+See [renew a TLS certificate for GOV.UK](/manual/renew-a-tls-certificate.html).
 
 ## Production www.gov.uk certificate
 
-The TLS certificate for www.gov.uk is managed by Fastly. They will open a support
-ticket when the certificate is due for renewal. This ticket will be picked up by
-GOV.UK Replatforming, who will co-ordinate with Fastly to renew the
-certificate.
+The TLS certificate for www.gov.uk is managed by Fastly. If any additional
+verification of domain ownership is needed for renewal (for example if Fastly
```

Comment:
This reads as though Fastly will only open a support ticket with us to do the renewal if additional verification is required. If not, Fastly will just do the renewal without any input from us? Is that correct?